variables.tf

```diff
@@ -52,4 +52,68 @@ variable "cluster_issuer_yaml" {
 variable "additional_set" {
   description = "Additional sets to Helm"
   default     = []
-}
\ No newline at end of file
+}
+
+variable "solvers" {
+  description = "List of Cert manager solvers"
+  #type        = list(map(any))
+  type        = any
+  default = [{
+    http01 = {
+      ingress = {
+        class = "nginx"
+      }
+    }
+  }]
+  #
```

Comment:
Good idea to add this into README.md. Like: how to work with multiple solvers.

Or you can remove it, i will update readme by myself

Reply:
```
# Example from https://github.com/terraform-iaac/terraform-kubernetes-cert-manager/issues/5
  #
  # [
  #   {
  #     dns01 = {
  #       route53 = {
  #         region  = "us-east-1"
  #         ambient = "true"
  #       }
  #     },
  #     selector = {
  #       dnsZones = [
  #         "internal.example.com"
  #       ]
  #     }
  #   },
  #   {
  #     dns01 = {
  #       cloudflare = {
  #         email = "user@example.com"
  #         apiKeySecretRef = {
  #           name = "cloudflare-api-key-secret"
  #           key  = "API"
  #         }
  #       },
  #     },
  #     selector = {
  #       dnsZones = [
  #         "public.example.com"
  #       ]
  #     }
  #   },
  #   {
  #     http01 = {
  #       ingress = {
  #         class = "nginx"
  #       }
  #     }
  #   }
  # ]
  #
```

I mean this part. We can delete this also.

Reply:
```
# Example from https://github.com/terraform-iaac/terraform-kubernetes-cert-manager/issues/5
  #
  # [
  #   {
  #     dns01 = {
  #       route53 = {
  #         region  = "us-east-1"
  #         ambient = "true"
  #       }
  #     },
  #     selector = {
  #       dnsZones = [
  #         "internal.example.com"
  #       ]
  #     }
  #   },
  #   {
  #     dns01 = {
  #       cloudflare = {
  #         email = "user@example.com"
  #         apiKeySecretRef = {
  #           name = "cloudflare-api-key-secret"
  #           key  = "API"
  #         }
  #       },
  #     },
  #     selector = {
  #       dnsZones = [
  #         "public.example.com"
  #       ]
  #     }
  #   },
  #   {
  #     http01 = {
  #       ingress = {
  #         class = "nginx"
  #       }
  #     }
  #   }
  # ]
  #
```

I mean this part. We can delete this also.